web/src/components/hero.tsx

```diff
@@ -3,35 +3,36 @@ import { css } from "@emotion/core";
 import BackgroundImage from "./background-image";
 import theme from "../utils/theme";
 
-const hero = (height: string) => css`
+const hero = css`
+	position: relative;
 	min-height: 400px;
-	height: ${height};
 	padding-top: 13rem;
+	padding-bottom: 7rem;
 	color: #ffffff;
+	overflow-wrap: break-word;
 
 	h2 {
 		font-size: 4rem;
 	}
 
 	@media screen and (max-width: 800px) {
 		padding-top: 8rem;
+		padding-bottom: 2rem;
 	}
 `;
 
-const image = (height: string) => css`
+const image = css`
 	position: absolute;
 	top: 0;
-	left: 0;
 	width: 100%;
-	height: ${height};
-	min-height: 350px;
+	height: 100%;
 	z-index: -1;
 `;
 
-const scrollButton = (height: string) => css`
+const scrollButton = css`
 	display: block;
 	position: absolute;
-	top: calc(${height} - 1.5rem);
+	top: calc(100% - 1.5rem);
```

Comment:
```suggestion
	bottom: -1.5rem;
```
I think this would be the same?

Reply:
Oh yeah, good catch! 😃